{"status":"FAIL","msg":"FAIL:\n Uncaught: { message: \"Expected a Test262Error to be thrown but no exception was thrown at all\" }\n    at ../../test262/harness/assert.js:101\n    at ../../test262/test/language/statements/class/dstr/gen-meth-static-ary-ptrn-elem-id-init-throws.js:76\n\n","path":"../../test262/test/language/statements/class/dstr/gen-meth-static-ary-ptrn-elem-id-init-throws.js","memory_kb":0,"duration":59912878}